weasyl/login.py

```diff
@@ -289,3 +291,19 @@ def get_account_verification_token(email=None, username=None):
         statement = statement.where(logincreate.c.login_name == username)
 
     return d.engine.execute(statement).scalar()
+
+
+def check_email_blacklist(address):
```

Comment:
`is_email_blacklisted` might be a better name, since `check` could suggest (and means, elsewhere in the project) throwing an exception.